src/components/timeline/Timeline.scss

```diff
@@ -0,0 +1,47 @@
+#conferencetimeline {
```

Comment:
Let's steer away from IDs.

Also, Let's use hyphens in work separations #readibility